MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -20,12 +20,18 @@ final class FeedViewModel {
 	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onShowError: Observer<String>?
+	var onDismissError: Observer<Void>?
```

Instead of two closures to manage the error state, a simpler solution would be to use just one such as `onErrorStateChange: Observer<String?>`. 

Just like the `loading state` is controlled with a single closure instead of two.

So, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange(.none)` to hide the error before loading the feed. And when there's an error: `onErrorStateChange(Localized.Feed.errorMessage)`.